src/main/java/io/pivotal/cla/data/repository/CorporateSignatureRepository.java

```diff
@@ -32,7 +32,7 @@
 
 	default CorporateSignature findSignature(String claName, Collection<String> organizations, Collection<String> emails) {
 		PageRequest pageable = new PageRequest(0, 1);
-		List<String> emailDomains = emails == null ? Collections.emptyList() : emails.stream().map( e-> e.substring(e.lastIndexOf("@") + 1)).collect(Collectors.toList());
+		List<String> emailDomains = emails == null ? Collections.singletonList("") : emails.stream().map( e-> e.substring(e.lastIndexOf("@") + 1)).collect(Collectors.toList());
```

Comment:
What exactly would this be needed for? We cannot add NOT NULL here because it is possible for the email to be null and only have the github organization provided.

> What about the other IN keyword for gitHubOrganization?
 
Thanks! Fixed via af0250253122ff79a9f7e04dc654c418f1ac27ca